### PR TITLE
Cabbage, Red Onion and related salads.

### PR DESCRIPTION
**Pull request by FluffiestFloof**
## About the PR

This adds the following:
- Cabbage seeds, plant and produce. It is available in the MegaSeed.
- Brings back Red Onion into the MegaSeed so it can be grown and used to make salads.
- Adds Coleslaw and two salads: Caesar and Kimchi.

The Coleslaw and Kimchi sprites were made by Kurzaen on discord after request.

**Media**

ss (2023-02-18 at 10 42 32)

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added Cabbage, Red Onion and 3 new salads.

### CODE REVIEW

**Review by moonheart08 — CHANGES_REQUESTED** [review]
This kinda needs immediate use before I'd like to add it. Maybe add coleslaw and a salad?

**Comment by FluffiestFloof** [comment]
> This kinda needs immediate use before I'd like to add it. Maybe add coleslaw and a salad?

Added two salads and coleslaw.
Edited title and description of PR to reflect the changes.

File: Resources/Prototypes/Recipes/Cooking/meal_recipes.yml [hunk 1]
```diff
@@ -982,6 +982,45 @@
...
+  solids:
+    FoodBowlBig: 1
+    FoodOnionRedSlice: 1
+    FoodBreadPlainSlice: 1
+    FoodCheeseSlice: 1
+    FoodCabbage: 1
+
+- type: microwaveMealRecipe
+  id: RecipeKimchiSalad
+  name: caesar salad recipe
```
Comment (lapatison) [inline Resources/Prototypes/Recipes/Cooking/meal_recipes.yml:1014]:
```suggestion
  name: kimchi salad recipe
```

Comment (FluffiestFloof) [reply Resources/Prototypes/Recipes/Cooking/meal_recipes.yml:1014]: Damn I'm dumb, thanks

**Review by moonheart08 — APPROVED** [review]

**Pull request merged**